src/models/bridge-data-filter.model.ts

```diff
@@ -7,8 +7,8 @@ export class BridgeDataFilterModel {
   }
 
   isMethodCall(callData: string) {
-    return callData.startsWith(this.abiEncodedSignature) ||
-      callData.startsWith('0x' + this.abiEncodedSignature);
+    const methodSignature = callData.slice(0, 10);
```

Comment:
It was previously accepting `0xabab` and `abab` if the encodedSignature was `abab`. Why are we no longer accepting it?

Reply:
Because a pegout has only '0x' as the data. And all the other transaction types match 'startsWith('0x')', which is not what we need.
I don't think we will get transaction data without the prefix '0x', or will we? I didn't see any.